pkg/rancher-components/src/components/Form/Checkbox/Checkbox.vue

```diff
@@ -169,6 +169,12 @@ export default Vue.extend({
           addObject(this.value, this.valueWhenTrue);
         }
         this.$emit('input', this.value);
+      } else if (this.isString(this.valueWhenTrue)) {
+        if (this.isChecked) {
+          this.$emit('input', undefined);
+        } else {
+          this.$emit('input', this.valueWhenTrue);
+        }
```

Comment:
note to self: This gave me some pause but it's, in fact, the correct implementation. The entire containing code block is intended to "flip the checkbox value". I think this can be simplified in a refactoring in the future.